protostar/commands/test/cheatcodes/max_examples_cheatcode.py

```diff
@@ -0,0 +1,28 @@
+from typing import Callable
+
+from protostar.commands.test.test_config import TestConfig
+from protostar.commands.test.test_environment_exceptions import CheatcodeException
+from protostar.starknet.cheatcode import Cheatcode
+
+
+class MaxExamplesCheatcode(Cheatcode):
+    def __init__(
+        self,
+        syscall_dependencies: Cheatcode.SyscallDependencies,
+        test_config: TestConfig,
+    ):
+        super().__init__(syscall_dependencies)
+        self.test_config = test_config
+
+    @property
+    def name(self) -> str:
+        return "max_examples"
```

Comment:
```suggestion
        return "max_fuzz_examples"
```
Not sure, maybe would be easier to interpret the purpose

Reply:
I have mixed feelings about this -- originally I wanted to reserve space here for extending usage of this cheatcode in some future, unknown nature, test modes. This cheatcode also does work in standard mode -- it's essentially a no-op there, but it works; so that you can slap `max_examples` in `__setup__` in a suite with mixed standard and fuzz tests.